address/setup.cfg

```diff
@@ -0,0 +1,2 @@
+[flake8]
+max-line-length=120
```

Comment:
Did we not have a default one in the main hail directory?

Reply:
I copied this from batch

Reply:
Let's remove all the extraneous setup.cfg's as a separate PR